### ISSUE TITLE
The evaluation score of aloe in the CLEVRER validation set

### ISSUE THREAD

**Issue by ExpressStag001** [pairlab/SlotFormer#14]
Dear Authors,

Thank you for your excellent work on this paper!

I am currently attempting to replicate the Aloe project in your framework using the CLEVRER dataset. However, the official website of the CLEVRER dataset is currently not accessible, so I am unable to upload the test set results to obtain the score.

I would like to know whether the score in your paper is based on the test set or the validation set, or if you could provide the validation set score ( CLEVRER dataset based on Aloe* ).

This will be of great help to my replication work. I would be extremely grateful if you could provide it. Thank you very much for your time and for your valuable contribution to the community.

**Comment by Wuziyi616** [comment]
Thank you for your interest in our work. We used the test set. We also noticed that the official CLEVRER website was down when we were developing SlotFormer. After contacting the authors, we used the eval server hosted on [another website](https://eval.ai/web/challenges/challenge-page/667/leaderboard/1813). Unfortunately, we don't have the results on the eval set and it would be hard to test it as it's been 4 years.

**Comment by ExpressStag001** [comment]
Thank you very much for your response.

We noticed that the eval server hosted you mentioned for CLEVRER also stopped providing scoring services for the test set on January 17, 2026.

In our follow-up work, we plan to strictly follow the instructions in **clevrer.md** to train the Aloe model (using the precomputed slots you provided) and then use this trained Aloe model to evaluate on the validation set, establishing a new baseline for CLEVRER.

We intend to release these changes and our reproduction results in our GitHub repository in the future, hoping that this can be helpful to other researchers interested in Aloe or the CLEVRER dataset.

Thank you again for your contributions to the OCL field.
